tests/rgw/test_s3.py

```diff
@@ -65,16 +66,17 @@ def setup_s3_tests(client_node, rgw_node, config, build):
     repo_url = "https://github.com/ceph/s3-tests.git"
     client_node.exec_command(cmd="git clone -b {branch} {repo_url}".format(branch=branch, repo_url=repo_url))
 
-    log.info("Running bootstrap")
     if build.startswith('4'):
+        pkgs = ["python2-virtualenv", "python2-devel", "libevent-devel", "libffi-devel", "libxml2-devel",
```

Comment:
would suggest this list to be placed outside if-else block.

Reply:
these pkgs list is needed only for rhcs 4 and not for rhcs 3. but to think of it we need to address for rhcs 5 too. i will check on it.

Reply:
i am not sure how it works for `pacific`. we can address this when we see the problem.